src/bp/core/app/server.ts

```diff
@@ -352,10 +357,12 @@ export class HTTPServer {
     this.adminRouter.setupRoutes(this.app)
     await this.botsRouter.setupRoutes(this.app)
     this.internalRouter.setupRoutes()
+    this.messagingRouter.setupRoutes()
 
     this.app.use('/assets', this.guardWhiteLabel(), express.static(resolveAsset('')))
 
     this.app.use('/api/internal', this.internalRouter.router)
+    this.app.use(BASE_API_PATH, this.messagingRouter.router)
```

Comment:
Don't you think it would make more sense if the endpoint was under `/messaging` on something similar?